themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1792,6 +1791,12 @@ html body.tc-body.tc-single-tiddler-window {
 	width: 100%;
 }
 
+.tc-controlpanel svg.tc-image-button{
+	width: 1em;
+	height: 1em;
```

Comment:
@pmario, I believe that was so beforehand. It's nothing I have deliberately touched, just something I experimented with to find the right spot for fixing the oversized popup contents.

We can't easily touch that (not in this context) because things like tiddler icons also depend on that, e.g. the cp cogwheel.